packages/beta/package.json

```diff
@@ -7,6 +7,7 @@
   "main": "dist/index.js",
   "types": "dist/src/index.d.ts",
   "version": "5.7.4",
+  "version": "5.8.0",
```

Comment:
do not modify version file manually (!).
lerna does this automatically based on commit message